src/python/WMCore/ReqMgr/Service/Request.py

```diff
@@ -375,25 +375,32 @@ def _retrieveResubmissionChildren(self, request_name):
 
     def _handleNoStatusUpdate(self, workload, request_args):
         """
-        only few values can be updated without state transition involved
-        currently 'RequestPriority' and 'total_jobs', 'input_lumis', 'input_events', 'input_num_files'
+        For no-status update, we only support the following parameters:
+         1. RequestPriority
+         2. Global workqueue statistics, while acquiring a workflow
         """
-        if 'RequestPriority' in request_args:
+        if 'RequestPriority' in request_args and len(request_args) == 1:
```

Comment:
Shouldn't this regardless number of request_args?

Reply:
Maybe :) The problem is that we would allow updating the request parameters at any point in the lifetime of a request. With this change, it will fail and only allow RequestPriority. If we want to change more parameters, then the only moment is during assignment.